tests/unit/test_charm.py

```diff
@@ -114,6 +99,24 @@ def _set_other_node_api_address_in_peer_relation(self, relation_id: int, unit_na
             key_values=key_values,
         )
 
+    def _set_ca_certificate_secret(self, private_key: str, certificate: str) -> None:
+        """Set the certificate secret in the peer relation."""
```

Comment:
Did this docstring come from the copy paste? Should change it here and there if so.